### PR TITLE
Immutable settings

### PR DESCRIPTION
**Pull request by bidoubiwa**
Using the custom settings to give the desired settings they will be added before documentation addition on every scrap

The error handling on the settings should stay there until the Python SDK has a proper error handler. 

fixes: #14

### CODE REVIEW

File: scraper/src/meilisearch_helper.py [hunk 1]
```diff
@@ -101,11 +102,16 @@ class MeiliSearchHelper:
...
         self.meilisearch_client = meilisearch.Client(host_url, api_key)
         self.__delete_and_create_index(index_uid)
         self.meilisearch_index = self.__delete_and_create_index(index_uid)
-        self.meilisearch_index.update_settings(MeiliSearchHelper.SETTINGS)
+        settings = {**MeiliSearchHelper.SETTINGS, **custom_settings}
+        try:
+            self.meilisearch_index.update_settings(settings)
+        except Exception:
+            print("MeiliSearchApiError: Settings are badly formatted. ")
+            sys.exit()
```
Comment (curquiza) [inline scraper/src/meilisearch_helper.py:114]: No need to try/catch the Exception :)
For the moment the python package will raise an exception, even if it's not the best exception in the world 😉 The python package will be upgraded soon.
[Algolia does not try/catch at this point](https://github.com/algolia/docsearch-scraper/blob/5ce1d095bb76b0aca87d47dc8f7e045d2cf30f44/scraper/src/algolia_helper.py#L18), keep it simple 🙂

Comment (bidoubiwa) [reply scraper/src/meilisearch_helper.py:114]: If I remove the sys.exit() the scrape will continue ignoring the fact that the settings were not taken into account. Do we want that?

Comment (curquiza) [reply scraper/src/meilisearch_helper.py:114]: @bidoubiwa No, it will not continue: raising an exception stops the process.

Comment (bidoubiwa) [reply scraper/src/meilisearch_helper.py:114]: You'r right. I thought I tried it and it did not raise anything but I probably made an error during my test. 
Removing it!

File: scraper/src/meilisearch_helper.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 Wrapper on top of the MeiliSearch API client"""
 
 import meilisearch
+import sys
```
Comment (curquiza) [inline scraper/src/meilisearch_helper.py:5]:
```suggestion
```

**Pull request merged**